diff --git a/CMakeLists.txt b/CMakeLists.txt
index cb697d0..cd4b06c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -365,9 +365,9 @@ endif()
 
 if(BUILD_GENERATORS)
     # Code generation
-    add_h3_executable(generateBaseCellNeighbors src/apps/miscapps/generateBaseCellNeighbors.c ${APP_SOURCE_FILES})
-    add_h3_executable(generateNumHexagons src/apps/miscapps/generateNumHexagons.c ${APP_SOURCE_FILES})
-    add_h3_executable(generateFaceCenterPoint src/apps/miscapps/generateFaceCenterPoint.c ${APP_SOURCE_FILES})
+    add_h3_executable(generateBaseCellNeighbors src/apps/miscapps/generateBaseCellNeighbors.c ${ALL_SOURCE_FILES})
+    add_h3_executable(generateNumHexagons src/apps/miscapps/generateNumHexagons.c ${ALL_SOURCE_FILES})
+    add_h3_executable(generateFaceCenterPoint src/apps/miscapps/generateFaceCenterPoint.c ${ALL_SOURCE_FILES})
 
     # Miscellaneous testing applications - generating random data
     add_h3_executable(mkRandGeo src/apps/testapps/mkRandGeo.c ${APP_SOURCE_FILES})
