diff --git a/src/apps/miscapps/generateFaceCenterPoint.c b/src/apps/miscapps/generateFaceCenterPoint.c
index 68752ef..6f0b168 100644
--- a/src/apps/miscapps/generateFaceCenterPoint.c
+++ b/src/apps/miscapps/generateFaceCenterPoint.c
@@ -26,7 +26,7 @@
 /**
  * Generates and prints the faceCenterPoint table.
  */
-static void generate() {
+void generate() {
     printf("static const Vec3d faceCenterPoint[NUM_ICOSA_FACES] = {\n");
     for (int i = 0; i < NUM_ICOSA_FACES; i++) {
         GeoCoord centerCoords = faceCenterGeo[i];
